ethereum/spec/src/main/java/tech/pegasys/teku/spec/datastructures/state/beaconstate/versions/capella/BeaconStateSchemaCapella.java

```diff
@@ -29,23 +30,29 @@
 import tech.pegasys.teku.spec.config.SpecConfig;
 import tech.pegasys.teku.spec.config.SpecConfigCapella;
 import tech.pegasys.teku.spec.datastructures.execution.versions.capella.ExecutionPayloadHeaderSchemaCapella;
+import tech.pegasys.teku.spec.datastructures.state.PendingAttestation;
 import tech.pegasys.teku.spec.datastructures.state.SyncCommittee;
 import tech.pegasys.teku.spec.datastructures.state.beaconstate.BeaconStateSchema;
 import tech.pegasys.teku.spec.datastructures.state.beaconstate.common.AbstractBeaconStateSchema;
 import tech.pegasys.teku.spec.datastructures.state.beaconstate.common.BeaconStateFields;
 import tech.pegasys.teku.spec.datastructures.state.beaconstate.versions.altair.BeaconStateSchemaAltair;
+import tech.pegasys.teku.spec.datastructures.state.versions.capella.HistoricalSummary;
 
 public class BeaconStateSchemaCapella
     extends AbstractBeaconStateSchema<BeaconStateCapella, MutableBeaconStateCapella> {
   public static final int NEXT_WITHDRAWAL_INDEX = 25;
   public static final int NEXT_WITHDRAWAL_VALIDATOR_INDEX = 26;
+  public static final int HISTORICAL_SUMMARIES_INDEX = 27;
 
   @VisibleForTesting
   BeaconStateSchemaCapella(final SpecConfig specConfig) {
     super("BeaconStateCapella", getUniqueFields(specConfig), specConfig);
   }
 
   private static List<SszField> getUniqueFields(final SpecConfig specConfig) {
+    final HistoricalSummary.HistoricalSummarySchema historicalSummarySchema =
+        new HistoricalSummary.HistoricalSummarySchema();
+    new PendingAttestation.PendingAttestationSchema(specConfig);
```

Comment:
Why are we instantiating this PendingAttestation schema object here?

Reply:
because copy and paste :)